test-templates/src/fields.rs

```diff
@@ -395,7 +403,11 @@ macro_rules! __test_field {
             assert_eq!(BigUint::from(<$field>::MODULUS_MINUS_ONE_DIV_TWO), &modulus_minus_one / 2u32);
             assert_eq!(<$field>::MODULUS_BIT_SIZE as u64, modulus.bits());
             if let Some(SqrtPrecomputation::Case3Mod4 { modulus_plus_one_div_four }) = <$field>::SQRT_PRECOMP {
-                assert_eq!(modulus_plus_one_div_four, &((&modulus + 1u8) / 4u8).to_u64_digits());
+                let mut expected = ((&modulus + 1u8) / 4u8).to_u64_digits();
+                if <$field>::MODULUS_BIT_SIZE as u64 % 64 == 0 {
```

Comment:
Why is this necessary? Modulus_plus_one_div_four has always 255 bits at most, and so should fit in 4 limbs